src/repos/RequestBuilder.js

```diff
@@ -0,0 +1,13 @@
+
+class RequestBuilder {
```

Comment:
a helper to server as a gateway for outbound requests to add extra headers like authentication and parse the response as json no need to repeat those steps in every repo